README.md

```diff
@@ -41,9 +41,11 @@ Install SymEngine:
 
     cmake .
     make
+    make install
 
 This will configure and build SymEngine in the default Release mode with all
-code and compiler optimizations on.
+code and compiler optimizations on and then install it on your system.
+SymEngine can then be imported in your programs.
```

Comment:
Should I add `Note that you will have to link SymEngine during compilation` or should I remove this line?